addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -133,10 +134,6 @@ def _compile_directive_install(self, el, compile_context, indent):
             self._append_text(div, compile_context)
         return []
 
-    def _compile_directive_placeholder(self, el, compile_context, indent):
-        el.set('t-att-placeholder', el.attrib.pop('t-placeholder'))
-        return []
-
```

Comment:
? :thinking:

Reply:
It was wrongly used in E-commerce and I didn't see any other usage in Odoo so I decided to remove it